example/android/app/build.gradle

```diff
@@ -39,8 +42,8 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.amplitude_flutter_example"
-        minSdkVersion 16
-        targetSdkVersion 28
+        minSdkVersion 21
+        targetSdkVersion 33
```

Comment:
Why is targetSdkVersion 33 and compileSdkVersion 34?

Reply:
compileSdkVersion is usually set to the latest version while targetSdkVersion is a stable version. But 34 makes more sense as targetSdk [will need to be 34+](https://developer.android.com/google/play/requirements/target-sdk) for new apps and app updates by August 31, 2024. Updated at https://github.com/amplitude/Amplitude-Flutter/pull/205/commits/4b7b0032b2e25e5e134e335c3264d521ea7d6083